test/FsAutoComplete.Tests.Lsp/Utils/TextEdit.fs

```diff
@@ -382,25 +382,25 @@ module TextEdit =
       Some "Expected positive End.Character, but was negative"
     else if edit.Range.Start > edit.Range.End then
       Some "Expected Range.Start <= Range.End, but was Start > End"
-    else if edit |> doesNothing then
-      Some "Expected change, but does nothing (neither delete nor insert)"
+    // else if edit |> doesNothing then
+    //   Some "Expected change, but does nothing (neither delete nor insert)"
```

Comment:
Tests were failing because the diagnostic for the codefix may not actually contain text that it should replace at the cursor.

Reply:
Was able to fix this